marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/merging.sjs

```diff
@@ -65,14 +65,33 @@ function validateArtifact(artifact) {
 function defaultArtifact(artifactName) {
   const defaultPermissions = 'data-hub-common,read,data-hub-common,update';
   return {
-    artifactName,
-    collections: ['default-mastering'],
-    additionalCollections: [],
+    batchSize: 100,
+    threadCount: 1,
     sourceDatabase: dataHub.config.FINALDATABASE,
     targetDatabase: dataHub.config.FINALDATABASE,
-    provenanceGranularityLevel: 'coarse',
     permissions: defaultPermissions,
-    batchSize: 100
+    targetEntity: "Change this to a valid entity type name; e.g. Customer",
+    sourceQuery: "cts.collectionQuery('mastering-summary')",
+    collections: [
```

Comment:
@rdew, @rjrudin 
Should 'artifactName' be added to the "collections" array ?

Reply:
I defer to @ryanjdew on that.

Reply:
@srinathgit No. For merging, an empty array is best since the merging will set collections appropriately on a per dcoument basis.